lib/config/presets/internal/replacements.ts

```diff
@@ -3,7 +3,21 @@ import type { Preset } from '../types';
 export const presets: Record<string, Preset> = {
   all: {
     description: 'All replacements',
-    extends: ['replacements:jade-to-pug'],
+    extends: [
+      'replacements:jade-to-pug',
+      'replacements:flake8-pathlib-to-flake8-use-pathlib',
+    ],
+  },
+  'flake8-pathlib-to-flake8-use-pathlib': {
+    description: 'flake8-pathlib was renamed to flake8-use-pathlib',
+    packageRules: [
+      {
+        matchDatasources: ['pypi'],
+        matchPackageNames: ['flake8-pathlib'],
+        replacementName: 'flake8-use-pathlib',
+        replacementVersion: '0.2.1',
```

Comment:
I see that the old package had a version `0.1.4` released on 6th of april 2021, and the newer package has two releases:

- `flake8-use-pathlib 0.2.0` Apr 6, 2021
- `flake8-use-pathlib 0.2.1` Nov 10, 2021

Are you sure that people will want to upgrade from `flake8-pathlib 0.1.4` to the renamed package latest release `flake8-use-pathlib 0.2.1`?

Reply:
I was wondering about that.  Should I be setting the `replacementVersion` to the first release of the new package?

Reply:
I've downgraded that to 0.2.0.

Reply:
Generally we should upgrade to the first release of the new package, and a subsequent run of Renovate will do the upgrade from `0.2.0` -> `0.2.1`.